lib/vue/src/components/input/CInputConfirm.vue

```diff
@@ -61,39 +77,68 @@ export default {
     okDisabled: Boolean,
     cancelDisabled: Boolean,
     noPrompt: Boolean,
+    processing: Boolean,
+    isIconVisible: Boolean,
+
+    icon: {
+      type: Array,
+      default: () => ['far', 'trash-alt'],
+    },
 
     buttonClass: {
       type: String,
       default: '',
     },
+
+    iconClass: {
+      type: String,
+      default: '',
```

Comment:
the default class is probably 'text-danger', or in general the one that is used the most.